omero/developers/Web/CreateApp.txt

```diff
@@ -67,49 +69,70 @@ OR you could simply choose an existing location:
 Create and check out a new GitHub repository OR manually create a new directory
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
--  Login to your GitHub account homepage (e.g.
-   `<https://github.com/>`_\ <your-name>/) and click
+-  Login to your GitHub account homepage e.g.
+   `<https://github.com/>`_\<your-name>/) and click
    "New repository"
--  Enter the name of <your-app>, add a description and choose to add a
-   README.
 
--  Check out your new repository (into a new directory)
+-  Enter the name of your repository i.e. ``<organisation-app>`` e.g. ``omero-webtest``,
+   add a description and choose to add a ``README.rst``. Change the extension to ``.rst`` so
+   it can be rendered correctly on PyPI. Its contain should cover the goal of the app
+   and the configuration instructions.
+
+-  Check out your new repository into a new directory
 
    ::
 
-       $ git clone git@github.com:<your-name>/<your-app>.git
+       $ git clone git@github.com:<your-name>/<organisation-app>.git
 
 
 -  OR: If you have not used git to create your app directory above, then
 
    ::
 
-        $ mkdir <your-app>
+        $ mkdir <organisation-app>
 
--  In either case, you should now have a directory called ``your-app`` within
+-  In either case, you should now have a directory called ``organisation-app`` within
    a directory that is on your :envvar:`PYTHONPATH`.
 
+Make your app installable from PyPI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is not required but it is highly recommended to make your app
+installable from PyPI. Few files need to be added:
+
+- A ``MANIFEST.in`` file. This is needed in certain cases to package files not automatically included
+
+- A configuration file ``setup.cfg``: an ini file that contains option defaults for ``setup.py`` commands
+
+- A set-up file ``setup.py``. This is used to configure various aspects of the app and also is also a command line interface for packaging the app.
+
+
+See `Packaging and Distributing Projects <https://packaging.python.org/distributing/>`_ for more details. 
 
 Add the essential files to your app
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
--  Create an empty file ``<your-app>/__init__.py`` (NB: both double
-   underscores)
--  Create :file:`urls.py`
+-  Create a directory ``<organisation_appname>`` where to add the essential files 
```

Comment:
"where to add the essential files" -> "to add the essential files to"

Also, I'd make the "Note the underscore" in brackets at the end of the sentence rather than adding it as a new sentence